kms/attestations/README.rst.in

```diff
@@ -0,0 +1,19 @@
+# This file is used to generate README.rst
+
+product:
+  name: Google Cloud KMS API
+  short_name: Cloud KMS API
+  url: https://cloud.google.com/kms/docs/
+  description: >
+    The `Google Cloud KMS API`_ is a service that allows you to keep encryption
```

Comment:
just Cloud KMS API, throughout

Reply:
But can you check with PM about this blurb in case we need to sync with some standardized test?

Reply:
The description from https://cloud.google.com/kms/docs will be used.